manifests/o/Open-Shell/Open-Shell-Menu/4.4.190/Open-Shell.Open-Shell-Menu.locale.en-US.yaml

```
@@ -0,0 +1,26 @@
+# Created using wingetcreate 1.2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Open-Shell.Open-Shell-Menu
+PackageVersion: 4.4.190
+PackageLocale: en-US
+Publisher: Open-Shell
+PublisherUrl: https://open-shell.github.io/Open-Shell-Menu/
+PublisherSupportUrl: https://github.com/Open-Shell/Open-Shell-Menu/issues
+Author: Open-Shell
+PackageName: Open-Shell-Menu

```

```suggestion
PackageName: Open-Shell
```
This must match Control Panel.